src/app/pages/storage/volumes/volumerekey-form/volumerekey-form.component.ts

```diff
@@ -82,12 +90,18 @@ export class VolumeRekeyFormComponent  implements Formconfiguration {
   customSubmit(value) {
     this.loader.open();
 
-    return this.rest.post(this.resource_name + "/" + this.pk + "/rekey/", { body: JSON.stringify({passphrase: value.passphrase}) }).subscribe((restPostResp) => {
-      this.loader.close();
-      this.dialogService.Info(T("Re-keyed Pool"), T("Successfully re-keyed pool ") + value.name);
-
-      this.router.navigate(new Array('/').concat(
-        this.route_success));
+    return this.rest.post(this.resource_name + "/" + this.pk + "/rekey/", 
+      { body: JSON.stringify({passphrase: value.passphrase}) }).subscribe((restPostResp) => {
+        this.loader.close();
+        this.snackBar.open(T('Successfully re-keyed pool ' + value.name), T("Close"), {
```

Comment:
Moving the ) of the translation marker actually breaks the translation grabbing for this string